DataFormats/PatCandidates/interface/Muon.h

```diff
@@ -281,8 +281,10 @@ namespace pat {
       void  setSoftMvaValue(float softmva){ softMvaValue_ = softmva; }
 
       /// MC matching information
-      reco::MuonSimType simType() const { return simType_; }
-      reco::ExtendedMuonSimType simExtType() const { return simExtType_; }
+      /// returns reco::MuonSimType - we cast it to int to avoid issues in pyROOT
+      int simType() const { return simType_; }
+      /// reco::ExtendedMuonSimType - we cast it to int to avoid issues in pyROOT
+      int simExtType() const { return simExtType_; }
```

Comment:
This solution degrades type safety to serve a broken external use case.

I propose to introduce/add ```simTypeInt``` and ```simExtTypeInt``` with appropriate comments that it's to be used in FWLite environment where the scoped enums are not properly recognized.

Reply:
I don't think your solution will work well because the user needs to know that he has to use simTypeInt to be safe. I have personally rediscovered this problem 3 times, i.e. one tends to forget about it and use it as is. All these accessors are primarily used by endusers, i.e. there won't be many use cases where someone would really benefit from type safety. In other words the tradeoff is between a benefit that won't be used vs a real problem that affects a number of users.